src/BuildCheck.UnitTests/EditorConfig_Tests.cs

```diff
@@ -1038,7 +1038,7 @@ public void LongLines()
             properties);
     }
 
-    
+
```

Comment:
Is there a separate rule for "only ever one blank line"? I'd just delete these . . . but not a big deal, this PR is a strict improvement.

Reply:
[doesn't seem so, here is the list.](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/csharp-formatting-options#new-line-options)
interesting and kind of surprising.

Reply:
Looks like there's `dotnet_style_allow_multiple_blank_lines_experimental` but no concrete plans to make it non-`experimental`: https://github.com/dotnet/roslyn/issues/60539.